src/services/meetings/meetings-router.ts

```diff
@@ -1,107 +1,98 @@
 import { Router } from "express";
 import { StatusCodes } from "http-status-codes";
 import {
-    meetingView,
-    createMeetingValidator,
-    updateMeetingValidator,
-} from "./meetings-schema";
-
-import { Database } from "../../database";
+    getAllMeetings,
+    getMeetingById,
+    createMeeting,
+    updateMeeting,
+    deleteMeeting,
+} from "./meetings-util";
 import RoleChecker from "../../middleware/role-checker";
 import { Role } from "../auth/auth-models";
 
 const meetingsRouter = Router();
 
-// GET endpoints: get all meetings, get specific meeting
-// POST endpoints: create a meeting
-// PUT endpoints: edit a meeting
-// DELETE endpoints: delete a meeting
-
-// get all events
 meetingsRouter.get(
     "/",
     RoleChecker([Role.enum.STAFF, Role.enum.ADMIN]),
-    async (req, res) => {
-        const meetings = await Database.MEETINGS.find();
-        const parsedMeetings = meetings.map((meeting) =>
-            meetingView.parse(meeting.toObject())
-        );
-        res.status(StatusCodes.OK).json(parsedMeetings);
+    async (req, res, next) => {
+        try {
+            const meetings = await getAllMeetings();
```

Comment:
why is this hoisted out? imo just use supabase get logic and work with that, no point moving a single function out